Modules/Core/include/mitkImageCast.h

```diff
@@ -64,6 +65,24 @@ namespace mitk
   template <typename TPixelType, unsigned int VImageDimension>
   extern void MITKCORE_EXPORT CastToItkImage(const mitk::Image * mitkImage, itk::SmartPointer<itk::VectorImage<TPixelType,VImageDimension> >& itkOutputImage);
 
+  template <typename TPixelType, unsigned int VImageDimension>
+  void CastToItkImageSingleComponent(const mitk::Image* mitkImage, itk::SmartPointer<itk::Image<TPixelType, VImageDimension> >& itkOutputImage, int component)
+  {
+    if (mitkImage->GetPixelType().GetNumberOfComponents() == 1) {
+      CastToItkImage(mitkImage, itkOutputImage);
+    } else {
+      itk::VectorImage<TPixelType, VImageDimension>::Pointer temp = itk::VectorImage<TPixelType, VImageDimension>::New();
```

Comment:
74:7: error: need ‘typename’ before ‘itk::VectorImage<TPixelType, VDimension>::Pointer’ because ‘itk::VectorImage<TPixelType, VDimension>’ is a dependent scope
       itk::VectorImage<TPixelType, VImageDimension>::Pointer temp = itk::VectorImage<TPixelType, VImageDimension>::New();
       ^~~